src/adaptations/device-layer/include/Weave/DeviceLayer/EventLoggingManager.h

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      Defines the Weave Device Layer EventLoggingManager object.
+ *
+ */
+
+#ifndef EVENT_LOGGING_MANAGER_H
+#define EVENT_LOGGING_MANAGER_H
+
+#include <Weave/Profiles/data-management/Current/DataManagement.h>
+
+namespace nl {
+namespace Weave {
+namespace DeviceLayer {
+namespace Internal {
+
+/**
```

Comment:
I'm a little unclear on how this comment relates to the intended purpose of the EventLoggingManager.